libs/execution_context/ebpf_core.c

```diff
@@ -1750,6 +1756,34 @@ _ebpf_core_get_pid_tgid()
     return ((uint64_t)ebpf_platform_process_id() << 32) | ebpf_platform_thread_id();
 }
 
+static uint64_t
+_ebpf_core_get_current_logon_id(_In_ const void* ctx)
+{
+    uint64_t logon_id = 0;
+
+    UNREFERENCED_PARAMETER(ctx);
+
+    if (!ebpf_is_preemptible()) {
+        return 0;
```

Comment:
Missing tracelogs? (I see instances of this omission in other error handling blocks as well)

Reply:
This is a helper function and may be used in a hot path. I think we should not add a trace here.

Reply:
Ah. If this is a 'benign' (and frequent) sort of an error, then skipping logging makes sense.  In such cases, a comment indicating this would be helpful.

If _not_ OTOH, then IMO logging this error would still make sense.  I don't know enough about what's involved here, so will defer to your judgement :-)

Reply:
On second thought, added trace here.